about.qmd

```diff
@@ -4,19 +4,16 @@ image: profile.jpg
 about:
   template: jolla
   links:
-    - icon: twitter-x
-      text: Twitter
-      href: https://twitter.com/Sheffield_R_
-    - icon: github
-      text: Github
-      href: https://github.com/SheffieldR
-    - icon: slack
-      text: Slack
-      href: https://sheffieldr.slack.com
-    - icon: mastodon
-      text: Mastodon
-      href: https://fosstodon.org/@sheffieldr
-
+      - icon: github
+        href: https://github.com/SheffieldR
+      - icon: calendar-event
+        href: https://www.meetup.com/sheffieldr-sheffield-r-users-group/
+      - icon: slack
+        href: https://sheffieldr.slack.com
+      - icon: twitter-x
```

Comment:
The twitter-x logo doesn't render for me locally but it's OK on the live site so I guess that's just a local issue for me.

For consistency, the indentation here should be reduced by 2 spaces.

Reply:
Will be removing twitter